src/provisioner/provisioner.py

```diff
@@ -94,19 +88,19 @@ def config_apply(solution_config_url: str, cortx_conf_url: str = None,
             Log.error(f'Unable to load {solution_config_url} url, Error:{e}')
 
         # Secrets path from config file
-        if cortx_conf.get('cortx>common>storage>local'):
-            CortxProvisioner._secrets_path = cortx_conf.get('cortx>common>storage>local')+CortxProvisioner._rel_secret_path
+        if tmp_conf.get('cortx>common>storage>local'):
```

Comment:
tmp_conf needs to be more meaningful. Does not highlight what does it container or mean. 

I feel cortx_conf is still better. You only need to add a comment that cortx_conf is temporary

Reply:
replaced with cortx_conf again.